web/satellite/src/components/account/billing/billingTabs/BillingHistoryShape.vue

```diff
@@ -0,0 +1,228 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="billing-history">
+        <div class="billing-history__common-info">
+            <span class="name-container" :title="historyItem.start">
+                <Calendar />
+                <p class="name_date">{{ historyItem.start }}</p>
+            </span>
+        </div>
+        <div class="billing-history__common-info">
+            <span class="name-container" :title="historyItem.status">
+                <CheckIcon class="checkmark" />
+                <p class="name_status">{{ historyItem.status }}</p>
+            </span>
+        </div>
+        <div class="billing-history__common-info">
+            <div class="name-container" :title="historyItem.amount">
+                <p class="name_amount">{{ historyItem.amount }}</p>
+            </div>
+        </div>
+        <div v-if="historyItem.link" class="billing-history__common-info">
+            <a href="historyItem.link" download>
+                <div class="name-container" :title="historyItem.link">
+                    <p class="name_download">Invoice PDF</p>
+                </div>
+            </a>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+import { PaymentsHistoryItem } from '@/types/payments';
+
+import { AccessGrant } from '@/types/accessGrants';
+import CheckIcon from '@/../static/images/billing/check-green-circle.svg';
+import Calendar from '@/../static/images/billing/calendar.svg';
+
+// @vue/component
+@Component({
+    components: {
+        Calendar,
+        CheckIcon,
+    },
+})
+export default class BillingHistoryShape extends Vue {
```

Comment:
Why is this component called BillingHistoryShape?

Reply:
Vlist takes a dom shape as a property. So I named it BillinghistoryShape as it's passed into Vlist.